build.rs

```diff
@@ -0,0 +1,38 @@
+use std::process::Command;
+
+use chrono::Utc;
+
+const BUILD_INFO_FALLBACK: &str = "Unknown";
+
+fn run_command(program: &str, args: &[&str]) -> String {
+    if let Ok(o) = Command::new(program).args(args).output() {
+        let raw_output = String::from_utf8_lossy(&o.stdout).into_owned();
+        raw_output.trim().to_string()
+    } else {
+        BUILD_INFO_FALLBACK.to_string()
+    }
+}
+
+fn set_env(var: &str, value: String) {
```

Comment:
nit: better to take `value: &str` since this function does not need ownership of `value`.